fix : add URL to a missing link

jypark0407 opened the pull request:
In Relevant Links (line 54), the first URL text 'Using typeof' was missing a href value. I fixed it by adding the link to a FreeCodeCamp's Javascript Debugging curriculum that introduces typeof method. (
Debugging: Use typeof to Check the Type of a Variable : https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/debugging/use-typeof-to-check-the-type-of-a-variable)

* I'm a first-time open source contributor. If you have any feedback about my pull request, feel free to share it with me.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

jypark0407 committed:
fix : add URL to a missing link

In Relevant Links (line 54), the first URL text 'Using typeof' was missing a href value. I fixed it by adding the link to a FreeCodeCamp's Javascript Debugging curriculum that introduces typeof method. (
Debugging: Use typeof to Check the Type of a Variable : https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/debugging/use-typeof-to-check-the-type-of-a-variable)

* I'm a first-time open source contributor. If you have any feedback about my pull request, feel free to share it with me.

RandellDawson commented on guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/boo-who/index.md:
```diff
@@ -53,7 +53,7 @@ Uses the operator `typeof` to check if the variable is a boolean. If it is, it w
 #### Relevant Links
 
-*   <a>Using typeof</a>
+*   <a href ='https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/debugging/use-typeof-to-check-the-type-of-a-variable' target='_blank' rel='nofollow'>Using typeof</a>
```
Since there is already an MDN link for **typeof**, I suggest just removing the first bullet all together
```suggestion
```

jypark0407 replied:
Okay, didn't see the MDN link before. Thanks for pointing out!

RandellDawson reviewed (changes requested):
See my suggested change.

jypark0407 committed:
Update guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/boo-who/index.md

Remove the first bullet instead of adding the link (suggested by RandellDawson)

Co-Authored-By: Randell Dawson <5313213+RandellDawson@users.noreply.github.com>

RandellDawson commented:
Congrats @jypark0407 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

jypark0407 commented:
Thanks @RandellDawson ! I look forward to contributing more to this wonderful project :)
